guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -41,11 +41,44 @@ ifdef::satellite,katello[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
+# subscription-manager repos --disable '*'
+# subscription-manager repos \
+ifdef::satellite[]
+--enable={RepoRHEL8ServerSatelliteServerProductVersion} \
+endif::[]
+--enable={RepoRHEL8BaseOS} \
```

Comment:
I think the heading on line 40 should be `{RHEL}` instead of `{EL}` given these are the RHEL repo names. And the same on line 23.

Reply:
Technically, correct. But this is the 3.1 branch, which is not supported by Foreman anyways anymore, and it renders correctly for Satellite, so I went with the smaller diff ;)